docs/core-manage-asdf-vm.md

```diff
@@ -59,18 +59,12 @@ No match for _Operating System_ and _Installation Method_ selections. Please try
 
 ### --Git--
 
-Clone only the latest branch:
+Run the install script to `git clone` the latest release:
 
 ```shell
-git clone https://github.com/asdf-vm/asdf.git ~/.asdf --branch v0.7.8
-```
-
-Alternately, you can clone the whole repo and checkout the latest branch:
-
-```shell
-git clone https://github.com/asdf-vm/asdf.git ~/.asdf
-cd ~/.asdf
-git checkout "$(git describe --abbrev=0 --tags)"
+curl -o- https://raw.githubusercontent.com/asdf-vm/asdf/v0.7.8/install.bash | bash
+# or
+wget -qO- https://raw.githubusercontent.com/asdf-vm/asdf/v0.7.8/install.bash | bash
```

Comment:
I don't think we should adopt this pattern. I know it's popular but it has some risks and I don't think we should push our users in this direction.

See:

* https://sysdig.com/blog/friends-dont-let-friends-curl-bash/
* https://www.seancassidy.me/dont-pipe-to-your-shell.html
* https://news.ycombinator.com/item?id=12766049

Reply:
I agree with all of this. Given it is the install method of choice for Homebrew & OMZSH users I didn't think it was too much an ask, especially since others can still just `git clone`. Additionally, my understanding of prior discussion of package manager code in this repo was that we didn't want to maintain those either.